test/Mono.Linker.Tests.Cases/Libraries/DefaultLibraryLinkBehavior.cs

```diff
@@ -4,6 +4,9 @@
 namespace Mono.Linker.Tests.Cases.Libraries
 {
 	[SetupCompileAsLibrary]
+#if NETCOREAPP
+	[IgnoreTestCase("Requires better testing framework support when no files are produced")]
```

Comment:
Should at least create an issue for this....

Reply:
Made different change which makes this go away